test/parallel/test-process-setuid-setgid.js

```diff
@@ -57,7 +57,15 @@ if (process.getuid() !== 0) {
 
 // If we are running as super user...
 const oldgid = process.getgid();
-process.setgid('nobody');
+try {
+  process.setgid('nobody');
+} catch (err) {
+  if (err.message !== 'setgid group id does not exist') {
+    throw err;
+  } else {
```

Comment:
Nit: You don't need the `else` here because of the `throw` in the preceding block. I don't mind it, though.

Reply:
Took me a while to figure out why we won't need the `else` block, got it now :) Will keep it in mind for the future